src/Details.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+export default class Details extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      message:
+        "Hello , this will be the details page for each Movie and TV show"
+    };
+  }
+
+  componentDidMount = () => {
+    // setTimeout( function () { return this.setState({message: "Coming soon! :)"})} , 3000)
```

Comment:
You can delete the commented out code not to be confused in the future

Reply:
I have kept that on purpose since the `setTimeout` function works when written as arrow fn and not when written the conventional way

Reply:
`( ) => ` this is a anonymous function. For this reason you do not need to specify function before. You do that when your naming your function, but this one is anonymous one.